src/tls/keypair.h

```diff
@@ -610,7 +590,7 @@ namespace tls
       HASH(contents, contents_size, hash.data());
 
 #if CURVE_CHOICE_SECP256K1_BITCOIN
-      return verify_secp256k_bc(ctx_, sig, hash.data());
+      return verify_secp256k_bc(ctx_, sig, hash.data(), &c4_pub);
```

Comment:
We have `if (signature.size() != REC_ID_IDX + 1) return false` before every other call to `verify_secp256k_bc`, should it also be here, or rolled into `verify` itself?

Reply:
It makes more sense to have it in one place, thanks for noticing :)